### ISSUE TITLE
Gamma function?

### ISSUE THREAD

**Issue by aam-berg** [lindeloev/mcp#170]
With regards to "families and link functions", can you please add a gamma function?

**Comment by lindeloev** [comment]
Yes, custom families is on the drawing board for v0.5 after the upcoming v0.4. v0.4 may be out in about a month (mostly docs remain) but v0.5 may be several months later as things look now.
